crates/rust-analyzer/src/to_proto.rs

```diff
@@ -696,7 +703,18 @@ pub(crate) fn snippet_text_document_edit(
 ) -> Result<lsp_ext::SnippetTextDocumentEdit> {
     let text_document = optional_versioned_text_document_identifier(snap, file_id);
     let line_index = snap.file_line_index(file_id)?;
-    let edits = edit.into_iter().map(|it| snippet_text_edit(&line_index, is_snippet, it)).collect();
+    let outside_workspace_annotation = snap
+        .analysis
+        .is_library_file(file_id)
+        .unwrap()
```

Comment:
?

Reply:
`is_library_file` returns `Cancelable`, is there a better way to get to the bool than unwrapping? 😅 Assuming you mean the unwrap here

Reply:
I mean, ? Is a better way to get a bool:)

Reply:
Woah I think I really spaced out there